source/projects/private-voice-assistant/voice_remote_local_assistant.markdown

```diff
@@ -0,0 +1,35 @@
+---
+title: "Configuring a local voice assistant"
+---
+
+If you have Home Assistant Cloud, the voice assistant uses Home Assistant Cloud by default.
+This is because the Cloud version currently still performs better than the local version.
+The performance depends a bit on the hardware you run it on. To set up a fully local, private voice assistant, follow these steps.
+
+## Setting up a local voice assistant
+
+1. Install the add-ons to convert text into speech and vice versa.
+   * Under **Settings** > **Add-ons** > **Z-Wave JS**, install the {% my supervisor_addon addon="whisper" title="**Whisper**" %} and the {% my supervisor_addon addon="piper" title="**Piper**" %} 
```

Comment:
Z-Wave JS?

Reply:
Also, you don't need to go to settings/add-ons if you click a My link, as that deeplinks to the right location.

Reply:
I know. I'm always mentioning how to get there because I used My links a lot. The problem was then (and still is) for me that this way I don't learn where things are located within HA. And if I then look for something while I'm not coming from the documentation I struggle to find things. 
This way I can click on the My link for convenience but I still see the path where this is taking me.